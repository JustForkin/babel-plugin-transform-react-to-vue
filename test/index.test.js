const babel = require('babel-core')

const transform = input => {
  return babel.transform(input, {
    plugins: [require.resolve('../')]
  }).code
}

test('main', () => {
  expect(transform(`
  class Foo extends Component {}
  `)).toMatchSnapshot()
})
