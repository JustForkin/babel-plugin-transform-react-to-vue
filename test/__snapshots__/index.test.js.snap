// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `"var Foo = {};"`;
